e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep

```diff
@@ -85,6 +92,13 @@ param OperationalInsightsName string {
   }
 }
 
+param OperationInsightsLocation string {
+  default: 'eastus'
```

Comment:
What other region options do we have? Anything closer to the west coast?

Reply:
We do have support for west us region, I'll update to that.